packages/manager/apps/dedicated/client/app/ip/ip.module.js

```diff
@@ -41,6 +41,6 @@ angular
   .filter('ipFilterByService', ipFilterByService)
   .filter('ipFirewallRulePort', ipFirewallRulePort)
   .filter('ipPunycode', ipPunycode)
-  .run(/* @ngTranslationsInject:json ./translations */);
+  .run(/* @ngTranslationsInject:json ./translations ./../../../../../modules/vrack/src/vrack-associated-services/ipv6/translations */);
```

Comment:
Loaded the translations in the module itself. We cannot have this way of loading.